packages/db/test/test-helpers.ts

```diff
@@ -11,3 +14,31 @@ export function createDbTest(): Database {
 export function randomName(prefix: string): string {
   return `${prefix}-${Math.random().toString(36).substring(2, 9)}`
 }
+
+export function testAccountInputCreate(input: Partial<AccountInputCreate> = {}): AccountInputCreate {
```

Comment:
Implement these helper functions to make it more declarative, and it saves updating the test at dozens of places if we need to add a new required property.